### PR TITLE
Addressed typos in Updating Methods for the TS Databases

### PR DESCRIPTION
**Pull request by ndharms**
There were some typos and py2->py3 issues that crept up when I was trying to use previous updating methods to update the TS database with new TS training data. This PR addresses makes these fixes. Two more PRs will exist that will (1) provide an example script that can update the database automatically and (2) provide an updated database with new AutoTST numbers.

### CODE REVIEW

**Comment by ndharms** [comment]
@davidfarinajr can you look at this? Also, what are our thoughts on 3 out of 4 checks passing? The only check not passing is the coverage for the patch.

File: autotst/data/update.py [hunk 1]
```diff
@@ -67,10 +67,8 @@ def get_unknown_species(reactions, known_species):
...
     for i, reaction in enumerate(reactions):
         rmg_reaction = reaction.rmg_reaction
-        r1, r2 = rmg_reaction.reactants
-        p1, p2 = rmg_reaction.products
 
-        relavent_species = [r1, r2, p1, p2]
+        relavent_species = rmg_reaction.reactants + rmg_reaction.products
         relavent_labels = {}
 
         for rel_species in relavent_species:
```
Comment (davidfarinajr) [inline autotst/data/update.py:75]: If these are labeled species, then I think we should be using in initial map with the atom labels when checking isomorphism.

**Review by davidfarinajr — COMMENTED** [review]
Looks good.  I am not familiar with the `get_unknown_species` method in `update.py`, but if we are checking isomorphism with labeled species, we might want to include an initial map of the labeled atoms.

File: autotst/data/update.py (hunk 1 above)
Comment (ndharms) [reply autotst/data/update.py:75]: @rwest comment: there's an `generate_initial_map` option to most isomorphism methods that generates a mapping by matching labels, so you don't need to make your own map. Might be worth trying

Comment (ndharms) [reply autotst/data/update.py:75]: Okay, after looking through this a bit more, this chunk of the code is simply designed to find a list of species that need to be added to a dictionary containing both SMILEs strings as keys and RMG Molecule objects as values. This dictionary is then used to generate SMILEs string labels used for creating reaction strings (`r1 + r2 = p1 + p2`) when adding comments to the saved TS database. But good catch on if we need the labels or not.

**Comment by ndharms** [comment]
Okay, so all of the tests pass and overall coverage increased but the coverage for the path is low. Are we okay with merging this in anyway or should I add more testing for the patch?

**Comment by ndharms** [comment]
@davidfarinajr @rwest any complaints about the patch coverage? Otherwise, can we merge this in?

**Comment by davidfarinajr** [comment]
No complaints from me

**Comment by rwest** [comment]
Go ahead

**Pull request merged**